docs/user/exercises/quiz.rst

```diff
@@ -23,7 +23,7 @@ This section describes the process to:
 Create new quiz exercise
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- In the Artemis home page, click on Course Management button |CourseManagementButton|
+- In the Artemis home page, click on Course Management button |CourseManagementButton|.
 
 - Navigate into **Exercises** of a specific course clicking in the exercise button.
```

Comment:
Navigate into Exercises of a specific course "by" clicking "on" the exercise button